examples/demo/financial/correlations.py

```diff
@@ -169,16 +169,13 @@ def _create_data(self, *names):
         self.plotdata = plotdata
 
     def _selections_updated(self, event):
-        metadata_changed_event = event.new
+        new_metadata = event.object
         if self.corr_renderer is None:
             return
-        if (
-            not isinstance(metadata_changed_event, dict)
```

Comment:
We previously checked if the event was a dict, but now we replace `.` with `:`, ie we listen to `metadata:items` so that this check is no longer needed